### DISCUSSION TITLE
part-ii/2012-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#199] [Question comments]
Part II 2012 Algebraic Geometry: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2012-0/

**Comment by eha1234** [comment]
<details><summary>1st part</summary>

Riemann Hurwitz.
For why $X$ can't be embedded in $\mathbb{P}^1$, I used the formula for genus of plane curves, $g = (d-1)(d-2)/2$
</details>

<details><summary>2nd part</summary>

I used $x^2 = y^n -1$
</details>
